index.html

```diff
@@ -28,13 +34,13 @@
             <a class="content-banner-Btn" href="/items.html">구경하러 가기</a>
```

Comment:
```suggestion
            <a class="content-banner-btn" href="/items.html">구경하러 가기</a>
```
클래스명에는 규칙적으로 스네이크 케이스를 써주고 계시기 때문에 통일성있게 작성해 주시면 좋을 것 같습니다.